docs/install.md

```diff
@@ -0,0 +1,163 @@
+#Installation and Settings
+##Prerequisites
+The MongoDB module and sub-modules need some configuration to be useful. This
+guide assumes that :
+
+* a [MongoDB][download] 3.0 to 4.0 server instance is already installed, configured, and
+  available to connect to from the Drupal instance.
+* the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
```

Comment:
This is not the latest version when I pull from your repo ? It misses the footnotes you added.